docs/mesos/ManualInstallCalicoDockerContainerizer.md

```diff
@@ -11,71 +11,66 @@
-store, run the Docker daemon with the following additional parameter:
+If using etcd as a cluster store, for example, run the Docker daemon with the
+following additional parameter:
 
     --cluster-store=etcd://<ETCD HOST>:<PORT>
 
-Replacing `<ETCD HOST>:<PORT>` with the appropriate `IP/Hostname:Port`
+Replacing `<ETCD HOST>:<PORT>` with the appropriate `hostname:port`
 for your etcd cluster.
 
 ## Install and Run Calico
 It is very easy to install Calico to use with the
 Docker Containerizer.
 
-On each of your agents, you will need to download the `calicoctl` command-line tool:
-
-```
-curl -o /usr/bin/calicoctl -L https://github.com/projectcalico/calico-containers/releases/download/v0.18.0/calicoctl
-chmod a+x calicoctl 
-```
+1. On each Mesos Agents, download the `calicoctl` command-line tool:
 
-Now, you will need run the `calico/node` and
-`calico/node-libnetwork` containers.
+  ```
+  curl -o /usr/bin/calicoctl -L https://github.com/projectcalico/calico-containers/releases/download/v0.18.0/calicoctl
+  chmod a+x calicoctl
+  ```
 
-For production deployments, we recommend running the two
-containers as services. Visit our guide on [running Calico
-as a Service](../CalicoAsService.md) to learn how to do this.
+2. Launch the `calico/node` and `calico/node-libnetwork` containers.
 
-For test environments that you would like to get up and running
-quickly, you can run the `calicoctl node --libnetwork`:
+  For production deployments, we recommend running the two
+  containers as services. Visit our guide on [running Calico
+  as a Service](../CalicoAsService.md) to learn how to do this.
 
-```
-sudo ETCD_AUTHORITY=<ETCD HOST:PORT> ./calicoctl node --libnetwork
-```
+  For test environments that you would like to get up and running
+  quickly, you can launch the container with `calicoctl`:
 
-Again, be sure to set the ETCD_AUTHORITY to the correct `IP/Hostname:Port` for your etcd cluster.
+  ```
+  sudo ETCD_AUTHORITY=<ETCD HOST:PORT> ./calicoctl node --libnetwork
+  ```
 
-Once you've started the Calico services or you've run the `node` command,
-you should see two Calico containers running in Docker:
+  Again, be sure to set the ETCD_AUTHORITY to the correct `IP/Hostname:Port` for your etcd cluster.
 
-```
-$ docker ps
-CONTAINER ID        NAMES               IMAGE                           CREATED
-19263eda1810        calico-libnetwork   calico/node-libnetwork:v0.8.0   3 seconds
-f237fb21d357        calico-node         calico/node:v0.18.0             3 seconds
-```
+3. Ensure calico's services are running by checking for two Calico containers in Docker:
 
-## Configure Mesos for Docker Containerizer
+  ```
+  $ docker ps
+  CONTAINER ID        NAMES               IMAGE                           CREATED
+  19263eda1810        calico-libnetwork   calico/node-libnetwork:v0.8.0   3 seconds
+  f237fb21d357        calico-node         calico/node:v0.18.0             3 seconds
```

Comment:
(Again, if we change this to :latest, it should automatically adjust itself as part of the release process.)